apps/src/code-studio/components/progress/teacherPanel/SelectedStudentInfo.jsx

```diff
@@ -110,12 +117,8 @@ export default class SelectedStudentInfo extends React.Component {
           {paired && (
             <div>
               <div>{i18n.workedWith()}</div>
-              {navigator && (
-                <div key={navigator}>{i18n.partner({partner: navigator})}</div>
-              )}
-              {driver && (
-                <div key={driver}>{i18n.loggedIn({partner: driver})}</div>
-              )}
+              {isDriver && <div>{i18n.partner({partner: partner})}</div>}
+              {isNavigator && <div>{i18n.loggedIn({partner: partner})}</div>}
```

Comment:
There were supposed to be more changes to the UX here (e.g. to display the # of students in the pairing group) but the proposed strings didn't fit the space very well so I'm keeping the existing strings for now.